rhel8/profiles/abcd-low-inline.profile

```diff
@@ -0,0 +1,10 @@
+documentation_complete: true
+title: ABCD Low for Red Hat Enterprise Linux 8
```

Comment:
I think this can be now removed and abcd.yml as well. because we have merged the real ANSSI profiles to this branch.

Reply:
I agree. We have more like these in the test directory.